subscription_manager/tests/test_controller_get_all.py

```diff
@@ -0,0 +1,118 @@
+from datetime import date, datetime
+from unittest.mock import MagicMock
+
+import pytest
+
+from subscription_manager.controller import Controller
+from subscription_manager.dbhelper import DBHelper
+from subscription_manager.exceptions import WrongTypeException, SubscriptionException
+from subscription_manager.subscription import Subscription
+
+
+@pytest.fixture
+def subscriptions_db_list():
+    return [
+        {
+            "comment": "Generation date: 23/06/2020, 10:46:11",
+            "currency": "USD",
+            "frequency": "daily",
+            "name": "Youtube Music",
+            "owner": "Kevin",
+            "price": 8.3,
+            "start_date": datetime(2019, 5, 11, 0, 0),
+        },
+        {
+            "comment": "Generation date: 23/06/2020, 10:46:41",
+            "currency": "CNY",
+            "frequency": "monthly",
+            "name": "Sky Store",
+            "owner": "Mary",
+            "price": 12.97,
+            "start_date": datetime(2019, 4, 13, 0, 0),
+        },
+        {
+            "comment": "Generation date: 23/06/2020, 10:47:36",
+            "currency": "EUR",
+            "frequency": "weekly",
+            "name": "Spotify",
+            "owner": "Robin",
+            "price": 30.23,
+            "start_date": datetime(2019, 10, 19, 0, 0),
+        },
+    ]
+
+
+subscriptions_obj_list = [
+    Subscription(
+        owner="Kevin",
+        name="Youtube Music",
+        frequency="daily",
+        start_date=date(2019, 5, 11),
+        price=8.3,
+        currency="USD",
+        comment="Generation date: 23/06/2020, 10:46:11",
+    ),
+    Subscription(
+        owner="Mary",
+        name="Sky Store",
+        frequency="monthly",
+        start_date=date(2019, 4, 13),
+        price=12.97,
+        currency="CNY",
+        comment="Generation date: 23/06/2020, 10:46:41",
+    ),
+    Subscription(
+        owner="Robin",
+        name="Spotify",
+        frequency="weekly",
+        start_date=date(2019, 10, 19),
+        price=30.23,
+        currency="EUR",
+        comment="Generation date: 23/06/2020, 10:47:36",
+    ),
+]
+
+
+@pytest.fixture
+def mock_dbhelper(subscriptions_db_list) -> DBHelper:
```

Comment:
can you please move common texture into another file

Reply:
Thank you, I will make another pull request with these changes with all tests.